src/main/resources/application.properties

```
@@ -1,6 +1,5 @@
-spring.h2.console.enabled=true
-
-spring.datasource.driver-class-name=com.mysql.jdbc.Driver
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
+#spring.h2.console.enabled=true
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/chess
```

```suggestion
spring.datasource.url=jdbc:mysql://localhost:3307/chess
```

docker 외부 port는 3307로 되어있네요 :)

---

테스트 용도로 잠깐 바꾸고 다시 돌려놓는걸 놓쳤네요 ㅠ. 해당 부분도 반영하였습니다